charts/opentelemetry-kube-stack/examples/cloud-demo/values.yaml

```diff
@@ -0,0 +1,42 @@
+clusterName: demo
+opentelemetry-operator:
+  enabled: true
+defaultCRConfig:
+  presets:
+    otlpExporter:
+      enabled: true
+      endpoint: ingest.example.com:443
+      headers:
+        "access-token": "${ACCESS_TOKEN}"
+collectors:
+  daemon:
```

Comment:
Should we define the daemon as a known "collector" in the base values.yaml that just exports to debug and then a user simply says "here's the exporter I want"... OR do we give them an empty collectors map and then use these examples for how we expect users to apply? I think it's simpler to do the former as it gives users a one click install to begin running a good set of collectors